apps/server/src/modules/authorization/reference.loader.ts

```diff
@@ -31,6 +48,7 @@ export class ReferenceLoader {
 		this.repos.set(AllowedAuthorizationEntityType.School, { repo: this.schoolRepo });
 		this.repos.set(AllowedAuthorizationEntityType.Lesson, { repo: this.lessonRepo });
 		this.repos.set(AllowedAuthorizationEntityType.Team, { repo: this.teamsRepo, populate: true });
+		this.repos.set(AllowedAuthorizationEntityType.Submission, { repo: this.submissionRepo });
```

Comment:
Fehlt hier nicht noch das populate: true?

Reply:
Habs jetzt so implementiert, dass immer populiert wird. Also keine Option eingebaut. Hatten wir uns nach meinem Verständnis vorhin in der Runde drauf geeinigt.

Reply:
Ok, dann hab ich da wohl nicht aufgepasst. Finde es aber dadurch, wenn man nur auf diese Stelle schaut, etwas missverständlich. Ohne den Blick ins Repo könnte man denken, dass es nicht populated wird. Wäre hier ein konsistentes Vorgehen schöner?

Reply:
Die populate option im loader wurde leider auch fälschlich eingebaut und soll demnächst wieder entfernt werden. Daher finde ich es erstmal nicht soo kritisch.